src/arch/xtensa/CMakeLists.txt

```diff
@@ -355,6 +355,17 @@ if(MEU_PATH)
 	endfunction()
 
 	sof_add_meu_target(run_meu sof-${fw_name}.ri "")
+
+	sof_add_meu_target(run_meu_post_pv sof-${fw_name}-post-pv.ri "-mnpv true")
+
+	add_custom_target(
+		bin_post_pv
+		COMMAND ${CMAKE_COMMAND} -E copy sof-${fw_name}-post-pv.ri ${PROJECT_BINARY_DIR}/sof-${fw_name}-post-pv.ri
```

Comment:
Looks OK, but do we want to change the binary name?
Probably it allows for easy differentiation between binaries?

Reply:
That was the intention. You can build multiple version of binaries in the same dir, so it may be confusing if all would be name sof-plat.ri.

Reply:
But also driver exects only one name, so it may be better to leave it with 1 name
@mmaka1 @lgirdwood @tlauda What do you think?

Reply:
A postfix might help to maintain multiple binaries, signed with different keys and settings during the build process. The deployment process may rename the files if really required.

Reply:
@jajanusz does this mean we would have a ```make post_pv_install" that would install and rename the files or even create a link to the correct file ?

Reply:
@lgirdwood I thought by deplyment @mmaka1 meant some CI. I don't know if we want to clutter buildsystem with things like these, that are done manually & sporadically. Our `make install` is going to install only headers anyway so it doesn't care about these binaries.

Reply:
@jajanusz By the deployment I meant copying/renaming/linking-to the files as @lgirdwood described. Such actions are usually supported by the make install command.

Reply:
@mmaka1 Ok, thanks for info. I'll add that.

Reply:
@lgirdwood @mmaka1 I started working on this and it's going to be a mess. There at least two issues with combination of your requests.
1) Cmake is good only in supporting `make install` doing install-like target requires adding a lot of hard to read code (cmake that runs cmake scripts to some install target etc, something like our version scripts), maybe it's not that bad for 1 target, but it will become nightmare with more meu configs. Also making all these dependency chains meu->bin->install is error-prone.
2) Usage is unclear, when I do `make install` I install to `CMAKE_INSTALL_PREFIX`, when I do `make post_pv_install` do I override files? What will user expect? I don't even like `make bin` it's there because of it was in previous buildsystem and I didn't want to scary devs. TBH i'd remove `make bin` if `make install` is going to make its job and place some deep files in install folder.

We should focus on what we do on everyday basis, simple `make & install` flow, with not a much thoughts given about it. I don't care about things that are going to be made probably by the 1 (usually the same) person once in every 3 months. I suggest another approach to all that meu stuff - just make run_meu target check if there is file like meu.flags (or smth like that) and add/override flags used by meu. Flow will stay the same, u just change flags in that file (it's just replacement for cmdline args that we cannot have because make don't take args for targets).

@tlauda @mmaka1 @lgirdwood 
What do you think?

Reply:
Another option is adding cmake option like MEU_FLAGS but it's inconvienient to reconfigure the project to just sign binary differently.

Reply:
@jajanusz I think its's safe to have several ```make install``` rules for the binaries rules as long as people know what the results will be. e.g. make could print information at the end of the rule like
```
Installed SOF <production/debug/xxx> image version x,y,z signed with key KEY1
sof-apl.ri -> /lib/firmware/intel/sof-apl-vx.y.z-KEY1-prod.ri
```